Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo

```diff
@@ -0,0 +1,153 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Examples;
```

Comment:
I think this example is fine, but I would prefer to see a more concrete example that uses a variable (scheduled) demand load, and where the `dpMea` signal is measured, so that all of the controls can be validated in closed loop.
Right now, all of the example models are in open loop, which makes it difficult to potentially spot an issue with control sequences.

If you have a validation study coming down the pipeline, this suggestion can wait for that data, but otherwise, I would love to see an example that is more representative of the use of the plant.

Reply:
A more representative example has been implemented. 
![image](https://user-images.githubusercontent.com/82961137/129109310-c400e5d6-460b-4a3f-b86f-b5ffda87f2c9.png)
The cooling load now is scheduled to be constant before 6 AM and then varies in sine wave, reaching the peak value at 2 PM.

Also, the pressure difference of the demand side is now measured as the input for dpMea. A  closed loop is thus formed for the validation of the controls.